readme.md

```diff
@@ -1,4 +1,8 @@
 # Nimbus
 Nimbus is a .NET client library to add an easy to develop against experience against the Azure Service Bus brought to you by Damian Maclennan and Andrew Harcourt
 
+This repository is forked from [Nimbus API](https://github.com/NimbusAPI), bringing additional functionality on top of the functionality provided by Nimbus. This forkmaster branch is the default branch, and is still based on [Nimbus V2](https://github.com/NimbusAPI/Nimbus_v2), since version 3 has introduced changes such as at-most-once delivery guarantees, as opposed to at-least-once delivery guarantees.
```

Comment:
> This repository is ... bringing additional functionality
Can we please list at least one example of functionality to justify the existence of this repository?